src/main/java/ch/njol/skript/expressions/ExprItemsIn.java

```diff
@@ -54,6 +54,8 @@
 public class ExprItemsIn extends SimpleExpression<Slot> {
 	static {
 		Skript.registerExpression(ExprItemsIn.class, Slot.class, ExpressionType.PROPERTY, "[(all [[of] the]|the)] items ([with]in|of|contained in|out of) (|1¦inventor(y|ies)) %inventories%");
+
+		ExprLoopValue.registerLoopValueHandler(ExprItemsIn.class, (source, type) -> type.equalsIgnoreCase("item"));
```

Comment:
Contrasting to the other comments, I do think this one is needed because the return type is Slot, not ItemStack
![image](https://user-images.githubusercontent.com/29547183/187084609-7c70126b-9a22-41c3-83c2-4cbce429ebc0.png)

However, do you think this should be needed: should ExprLoopValue attempt converting from the return type of the loop expression to the requested type?

Reply:
I'm not sure - that would allow stuff like:
```vb
loop all players:
  broadcast loop-location
```
wouldn't it? I think it's better for us to force users to do `location of loop-player` instead

Reply:
Yea I agree on that. However, does that mean that we should add an loop value handler to every plural Slot expression?

Reply:
I don't really like the idea of adding a catch-all for `Slot` to ExprLoopValue directly (I'd like to keep it Bukkit free) so adding it to all of them may be our best option.

Reply:
One thing we *could* do is allow multiple classinfos to match the same user input pattern.
The current method would return the "first" whereas a new method could return all classinfos that match the user input. This would mean adding `items?` as a user input pattern to Slot.

Of course, this may be more hacky/strange than desirable.

Reply:
Decided to add a new registration process for direct type aliases :)